torch_geometric/explain/algorithm/captum_explainer.py

```diff
@@ -152,16 +158,14 @@ def forward(
             captum_model = CaptumModel(model, mask_type, index,
                                        self.model_config)
 
-        attribution_method = self.attribution_method(captum_model)
+        self.attribution_method_instance = self.attribution_method_class(
+            captum_model)
 
         # In captum, the target is the index for which
         # the attribution is computed.
-        if self.model_config.mode == ModelMode.regression:
-            target = None
-        else:
-            target = target[index]
+        target = index
```

Comment:
Can you give an example? What I have in mind is:

```
target = [0, 0, 0, 1, 0]
index = 2

# Applying the code:
target = target[index]  # target is now 0

# If this target (which is now 0) is used as an index in captum, 
# you'll get explanations for output index 0 instead of 2 (the original user-specified index).
```